ghga_connector/core/exceptions.py

```diff
@@ -193,7 +202,7 @@ class UserHasNoUploadAccessError(RuntimeError):
 
     def __init__(self, *, upload_id: str):
         message = (
-            "This user is not registered as the data submitter "
+            "User is not registered as a Data Submitter "
```

Comment:
"You are..." instead of "User is..."?

Reply:
I almost started replacing these references with "you" but wasn't sure. I'll tweak that